runtime/frequency/Cargo.toml

```diff
@@ -197,9 +205,6 @@ try-runtime = [
 	"pallet-democracy/try-runtime",
 	"pallet-utility/try-runtime",
 ]
-frequency = ["common-runtime/frequency"]
-frequency-rococo-testnet = ["common-runtime/frequency-rococo-testnet"]
-frequency-rococo-local = ["common-runtime/frequency-rococo-local"]
```

Comment:
I think this is what is causing the error in CI

Reply:
https://github.com/LibertyDSNP/frequency/blob/8f06976befb61df65e786c0aa0993c50813edaaa/runtime/frequency/Cargo.toml#LL200-L202